src/Text/Pandoc/Readers/Org/DocumentTree.hs

```diff
@@ -59,6 +61,9 @@ documentTree blocks inline = do
       , headlineContents = initialBlocks'
       , headlineChildren = headlines'
       }
+  where
+    commentLine :: Monad m => OrgParser m Blocks
+    commentLine = commentLineStart *> anyLine $> mempty
```

Comment:
I'd prefer `void $ ...` instead of `... $> mempty`: we just need the function's side-effects and don't use the return value.  Returning `()` makes that clear.

I assume this was copied over from Text.Pandoc.Readers.Org.Blocks? We do treat a comment line as a block in that file, but it makes less sense here.

Addendum: `commentLineStart` already has type `OrgParser m ()`, which allows to do `commentLineStart <* anyLine` and is probably even better than using `void`.

Reply:
Thanks, makes perfect sense. I just copied it over since I could not import it (cyclic import), I will change this in a minute.